azure/template.json

```diff
@@ -915,6 +927,18 @@
               {
                 "name": "PostcodeAnywhere:Key",
                 "value": "[parameters('PostcodeAnywhereKey')]"
+              },
+			  {
+                "name": "Zendesk:SnippetKey",
+                "value": "[parameters('ZendeskSnippetKey')]"
+              },
+			  {
+                "name": "Zendesk:SectionId",
+                "value": "[parameters('ZendeskSectionId')]"
```

Comment:
Might be best to name this ZendeskEmployerSectionId as they are different for employer and provider.

Reply:
👍